src/kubectl-helm-minikube/install.sh

```diff
@@ -311,6 +311,16 @@ if [ ${HELM_VERSION} != "none" ]; then
         echo '(!) Helm installation failed!'
         exit 1
     fi
+
+    # helm bash completion
+    helm completion bash > /etc/bash_completion.d/helm
```

Comment:
Is it possible to add some tests to validate this change?

Reply:
As I couldn't find any test as reference for the completion functionality (despite it is available for other commands, like, in this case, kubectl) let me check the [doc](https://github.com/devcontainers/cli/blob/main/docs/features/test.md) to get an idea of how to do it.

PS: @samruddhikhandale If you know about any other doc with additional info about feature testing, please, just let me know.

Reply:
@samruddhikhandale Test [added](https://github.com/devcontainers/features/pull/1080/commits/735d40d5a8e76e2f7da385e46d2c3d211a288b0b).  Just let me know if it looks good for you.